src/Desktop/ArcGISRuntime.Desktop.Samples/Samples/Map/ManageBookmarks/ManageBookmarks.xaml.cs

```diff
@@ -0,0 +1,159 @@
+﻿// Copyright 2016 Esri.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at: http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an 
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific 
+// language governing permissions and limitations under the License.
+
+using Esri.ArcGISRuntime.Mapping;
+using System.Linq;
+using System.Windows.Controls;
+
+namespace ArcGISRuntime.Desktop.Samples.ManageBookmarks
+{
+    public partial class ManageBookmarks
+    {
+     
+        public ManageBookmarks()
+        {
+            InitializeComponent();
+
+            // Create the UI, setup the control references and execute initialization. 
+            Initialize();
+        }
+
+        private void Initialize()
+        {
+            // Create new map with a basemap.
+            Map myMap = new Map(Basemap.CreateImageryWithLabels());
+
+            // Set the mapview, map property to the basemap.
+            MyMapView.Map = myMap;
+
+            // Create a set of predefined bookmarks; each one follows the pattern of:
+            // ~ Initialize a viewpoint pointing to a latitude longitude
+            // ~ Create a new bookmark
+            // ~ Give the bookmark a name
+            // ~ Assign the viewpoint
+            // ~ Add the bookmark to bookmark collection of the map
+            // ~ Add the bookmark name to the UI combobox for the user to choose from 
+
+            // Bookmark-1
+            Viewpoint myViewpoint1 = new Viewpoint(27.3805833, 33.6321389, 6000); 
+            Bookmark myBookmark1 = new Bookmark(); 
+            myBookmark1.Name = "Mysterious Desert Pattern"; 
+            myBookmark1.Viewpoint = myViewpoint1; 
+            MyMapView.Map.Bookmarks.Add(myBookmark1); 
+            bookmarkChooser.Items.Add(myBookmark1.Name);
```

Comment:
Add the Bookmark object to the combo (not just the name). This will simplify things when you need to get the selected Bookmark!
````
bookmarkChooser.Items.Add(myBookmark1); 
````